docs-source/content/userguide/introduction/introduction.md

```diff
@@ -62,6 +62,20 @@ AKS support and limitations:
   Infrastructure MDS data stores only when accessed through an OCI FastConnect.
 * Windows Server containers are not currently supported, only Linux containers.
 
+### VMware Tanzu Kubernetes Service
+
+Tanzu Kubernetes Grid (TKG) is a managed Kubernetes Service that lets you quickly deploy and manage Kubernetes clusters. The WebLogic Server Kubernetes
+Operator and Oracle WebLogic Sever are fully supported and certified on VMware Tanzu Kubernetes Grid Multicloud 1.1.3 (with vSphere 6.7U3).
+
+TKG support and limitations:
+
+* Both Domain in Image and Model in Image domain home source types are supported. Domain in PV is not supported.
+* VSphere CSI driver supports only volumes with Read-Write-Once policy. This does not allow writing stores on PV.  
+   * For applications requiring HA, use JMS and JTA stores in the database.
+* The ingress used for certification is NGINX, with MetalLB load balancer.
+* Public Internet access is allowed only on ports 80, 443.
```

Comment:
I am not sure about this item. I guess they have both restricted and unrestricted access to their services. Since we worked in a restricted environment we don't have access to other ports.

Reply:
So, would you suggest that I remove this bulleted item or should it be modified in some way?

Reply:
Yes we should remove this item.

Reply:
ok, thank you. Done.